src/js/components/homepageUpdate/WordOfTheDay/ErrorWordOfTheDay.jsx

```diff
@@ -7,13 +7,10 @@ import React from 'react';
 
 const ErrorWordOfTheDay = () => (
     <div className="word-of-the-day__error-message">
-        <picture>
-            <img
-                role="presentation"
-                src="../../../../img/homepage-word-of-the-day/Error-Placeholder.svg"
-                alt="" />
-        </picture>
-        Unable to load Word of the Day
+        <p className="word-of-the-day-paragraph-one">Something went wrong</p>
+        <p className="word-of-the-day-paragraph-two">Sorry, we're unable to load this content</p>
+        {/* eslint-disable-next-line no-return-assign */}
+        <button className="word-of-the-day-button" onClick={() => window.location = 'mailto:usaspending.help@fiscal.treasury.gov?subject=Word%20of%20the%20Day$20Error'}>Report this error</button>
```

Comment:
There's a typo in the last %20 so the subject shows up incorrectly

<img width="674" alt="Screen Shot 2022-10-27 at 8 11 19 AM" src="https://user-images.githubusercontent.com/121010/198280920-bd7d5414-4c19-4c2d-91e7-6f722b144b70.png">

Reply:
fixed